README.md

```diff
@@ -494,14 +526,11 @@ to find instruction and links to exact configuration files and final checkpoints
 ```
 
 ## Useful links
-- [Documentation](./docs)
+- [NNCF Documentation](./docs)
```

Comment:
Are you sure?

Reply:
Ok, moved it back. For me it's not crucial